.github/workflows/ubuntu.yml

```diff
@@ -39,6 +39,14 @@ jobs:
             elixir-version: '1.14.3'
             os: 'ubuntu-latest'
             version-type: 'strict'
+          - otp-version: '27.0'
+            os: 'ubuntu-24.04'
+            version-type: 'strict'
+          - otp-version: '26.0'
+            os: 'ubuntu-24.04'
+            version-type: 'strict'
+          - otp-version: '25.0'
```

Comment:
```suggestion
          - otp-version: '25'
```

Reply:
Applied in https://github.com/erlef/setup-beam/pull/271/commits/d2ad6440a8f7071bde167f803e52e9a9a9df10e8